save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/controller/HeartbeatController.kt

```diff
@@ -0,0 +1,39 @@
+package org.cqfn.save.orchestrator.controller
+
+import org.cqfn.save.orchestrator.model.AgentState.BUSY
+import org.cqfn.save.orchestrator.model.AgentState.ERROR
+import org.cqfn.save.orchestrator.model.AgentState.FINISHED
+import org.cqfn.save.orchestrator.model.AgentState.IDLE
+import org.cqfn.save.orchestrator.model.EmptyResponse
+import org.cqfn.save.orchestrator.model.Heartbeat
+import org.cqfn.save.orchestrator.model.HeartbeatResponse
+import org.cqfn.save.orchestrator.model.NewJobResponse
+import org.cqfn.save.orchestrator.model.TerminatingResponse
+import org.cqfn.save.orchestrator.service.AgentService
+import org.slf4j.LoggerFactory
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Mono
+
+@RestController
+class HeartbeatController(private val agentService: AgentService) {
+    private val logger = LoggerFactory.getLogger("org.cqfn.save.logback")
```

Comment:
```suggestion
    private val logger = LoggerFactory.getLogger(HeartbeatController::class.java)
```
Usually loggers have unique names per class, so that you can distinguish in logs, which class is producing data

Reply:
Fixed